packages/oc-template-handlebars-compiler/lib/compileServer.js

```diff
@@ -21,22 +22,32 @@ module.exports = (options, callback) => {
     stats
   });
 
-  compiler(config, (err, compiledServer) => {
-    if (err) {
-      return callback(err);
-    } else {
-      fs.ensureDirSync(publishPath);
-      fs.writeFile(
-        path.join(publishPath, publishFileName),
-        compiledServer,
-        err => {
-          callback(err, {
-            type: 'node.js',
-            hashKey: hashBuilder.fromString(compiledServer),
-            src: publishFileName
-          });
-        }
-      );
+  let compiledServer;
+
+  async.series(
```

Comment:
Why not `.waterfall()` instead? So that you could pass along the result (`server`) without having to rely/mutate external state (`let compiledServer`)